SYCL/ESIMD/api/functional/common.hpp

```diff
@@ -55,6 +55,40 @@ using shared_allocator = sycl::usm_allocator<DataT, sycl::usm::alloc::shared>;
 template <typename DataT>
 using shared_vector = std::vector<DataT, shared_allocator<DataT>>;
 
+// Provides verification that provided device has necessary aspects to interact
+// with current data type.
+//
+// Usage example:
+//
+// sycl::device device = queue.get_device();
+// if (should_skip_test_with(device)) {
```

Comment:
Minor comment: the example usage is incomplete. The call needs the data type.
In my opinion, the lines 60-69 should be simply removed.
Having the description at the lines 58-59 is sufficient.

Reply:
@vasilytric - please address review comments

Reply:
@kbobrovs I've applied this comment in this commit: 4977d372526f159aac3737caa810e2c99f6dc359